lxd/instance_exec.go

```diff
@@ -272,7 +274,7 @@ func (s *execWs) Do(ctx context.Context, op *operations.Operation) error {
 		}
 
 		// Make VM disconnections (shutdown/reboot) match containers.
-		if cmdErr == drivers.ErrExecDisconnected {
+		if cmdErr == drivers.ErrExecDisconnected || ctx.Err() != nil {
```

Comment:
I'm not sure about this bit, what was the reasoning here?

Reply:
Ah - I forgot to add a comment, sorry. This is so that if an admin boots a user from an exec session by deleting the operation, the exit code for the user is 129. I figured that would be a good error code to indicate disconnection?